message_test.go

```diff
@@ -78,7 +71,7 @@ func TestNewMessage(t *testing.T) {
 	}
 
 	if message.Gateway != 239 {
-		t.Errorf("Unexpected message gateway: %s", message.Gateway)
+		t.Errorf("Unexpected message gateway: %d", message.Gateway)
```

Comment:
In addition to your fix:
https://github.com/golang/go/wiki/CodeReviewComments#useful-test-failures